joblib/test/test_numpy_pickle.py

```diff
@@ -1011,8 +1018,12 @@ def test_lz4_compression_without_lz4(tmpdir):
     # Check that lz4 cannot be used when dependency is not available.
     fname = tmpdir.join('test.no.lz4').strpath
     data = 'test data'
-    compressor = 'lz4'
     with raises(ValueError) as excinfo:
-        numpy_pickle.dump(data, fname, compress=compressor)
+        numpy_pickle.dump(data, fname, compress='lz4')
+    excinfo.match('LZ4 in not installed. Consider installing it: '
+                  'http://python-lz4.readthedocs.io/')
+
+    with raises(ValueError) as excinfo:
+        numpy_pickle.dump(data, fname + '.lz4')
```

Comment:
`fname` already ends with `.lz4` in this test.

Reply:
indeed, fixed.